third_party/git-cmake-format/CMakeLists.txt

```diff
@@ -1,4 +1,3 @@
 load_git_package(git-cmake-format
     "https://github.com/gflegar/git-cmake-format.git"
-    "master")
-
+    "9fdc1553c525b3d7ce758892fe666078903a1b21")
```

Comment:
what is that? a special branch?

Reply:
The most recent commit in that repo. I changed all  repos to static commits to avoid problems Terry had with google test.